src/app/search/elastic-search-client.ts

```diff
@@ -18,7 +18,7 @@ import { Dockstore } from './../shared/dockstore.model';
 import { Client } from 'elasticsearch';
 
 export const ELASTIC_SEARCH_CLIENT = new Client({
-    host: Dockstore.API_URI + '/api/ga4gh/v1/extended',
```

Comment:
is ga4gh/v2 already in hotfix?

Reply:
Not yet I believe.

Reply:
I'll revert this change then.  It looks like the hotfix branch isn't using the v2 webservice either